docs/content/deployment/open-source.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Deploying Dagster with your infrastructure | Dagster Docs
+---
+
+# Deploying Dagster with your infrastructure
+
+<Note>
+  This section is applicable to Dagster Open Source. If you're looking for
+  Dagster Cloud, <a href="/dagster-cloud">click here</a>.
+</Note>
+
+Learn how to deploy and execute Dagster with these hands-on guides.
+
+---
+
+## Overview
+
+Get started with a look at Dagster's architecture, including a primer on deployment concepts. [Learn more](/deployment/overview).
+
+---
+
+## Concepts
+
+Learn about the concepts relevant to deploying Dagster.
+
+<ArticleList>
```

Comment:
Maybe we can add a link to the concepts page here, with a message like: "for an overview of Dagster's concepts, visit this page"?